eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/equipment/EquipmentRest.java

```diff
@@ -59,6 +63,18 @@ public Response readEquipment(@QueryParam("c") String equipment) {
 	public Response createEquipment(Equipment equipment) {
 		Equipment response = null;
 		try {
+            // Generate new numeric code if the requested code starts with @
+            if (equipment.getCode()!=null && equipment.getCode().startsWith("@")) {
+                String prefix = equipment.getCode().substring(1,equipment.getCode().length());
```

Comment:
`equipment.getCode().substring(1, equipment.getCode().length());` can be simplified to `equipment.getCode().substring(1);`

https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#substring(int)